### PR TITLE
Dimmer patches

### PR DESCRIPTION
**Pull request by pilino1234**
Fix 2 dimmer bugs

These are two separate commits since they are quite different, yet I feel that they are too similar to warrant opening separate PRs. Hope that's fine.

Discussion: https://forum.pilight.org/Thread-Dimlevel-text-in-WebGUI-0-and-not-updated

Commit messages: 

----
Webgui: Fix dimmer display bugs

This would cause the text field next to the slider to be
initialized to 0 at page load, even though the slider was showing the
correct dimlevel.

This commit fixes this by also assigning the dimlevel to the text field on
page load. Patch by @Niek.

Further, the slider-input field was editable, but without use (editing the
value has no effect). Therefore, display the field as disabled.

----

generic_dimmer: Fix inconsistency

The default max values were defined to different values in the checkValues and
createCode functions, causing the webgui to not transmit slider changes to
values above 10.
This commit fixes this by setting the default to 15 in both functions.

### CODE REVIEW

File: libs/pilight/protocols/generic/generic_dimmer.c [hunk 1]
```diff
@@ -75,7 +75,7 @@ static int createCode(JsonNode *code) {
 	int id = -1;
 	int state = -1;
 	int dimlevel = -1;
-	int max = 10;
+	int max = 15;
```
Comment (CurlyMoo) [inline libs/pilight/protocols/generic/generic_dimmer.c]: Better make this a global `define`

**Comment by pilino1234** [comment]
Done.

File: libs/pilight/protocols/generic/generic_dimmer.c [hunk 2]
```diff
@@ -75,7 +77,7 @@ static int createCode(JsonNode *code) {
 	int id = -1;
 	int state = -1;
 	int dimlevel = -1;
-	int max = 10;
+	int max = MAX_DIMMER_VALUE;
 	int min = 0;
```
Comment (CurlyMoo) [inline libs/pilight/protocols/generic/generic_dimmer.c]: The same counts of course for the minimal dimlevel :smile:

Comment (pilino1234) [inline libs/pilight/protocols/generic/generic_dimmer.c]: Hmm, forgot about that. Done :)

**Comment by CurlyMoo** [comment]
Thanks

**Comment by CurlyMoo** [comment]
You forgot the webgui version increase. Stupid me.

**Comment by pilino1234** [comment]
How to go about resolving this? I've amended the commit in by branch. I think the only way is to open a new PR since this one can't be updated any longer.

**Comment by CurlyMoo** [comment]
I know. A new PR is the only way.

**Comment by pilino1234** [comment]
What should the new PR include? Should the commits from this PR be reverted and added to a new PR, or just a commit to bump the webgui version?

**Comment by CurlyMoo** [comment]
Just the bump

**Comment by pilino1234** [comment]
Will do

**Pull request merged**